### PR TITLE
[Jetsurvey] Update to beta01 snapshot

### PR DESCRIPTION
**Pull request by florina-muntenescu**

### CODE REVIEW

File: Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyScreen.kt [hunk 1]
```diff
@@ -174,43 +177,47 @@ private fun TopAppBarTitle(
     )
 }
 
-@Suppress("DEPRECATION") // ConstraintLayout
-@OptIn(ExperimentalLayout::class)
+@Preview
+@Composable
+fun TAPPreview() {
```
Comment (JolandaVerhoef) [inline Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyScreen.kt:182]: Is this name correct? If you're referring to TopAppBar, maybe it should be TABPreview instead? But I'd suggest writing it out anyway.

**Review by JolandaVerhoef — APPROVED** [review]
One small nit, looks good otherwise

File: Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyScreen.kt (hunk 1 above)
Comment (chrisbanes) [reply Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyScreen.kt:182]: It should be `Preview<NameOfComposable>` according to convention (I think there's a lint rule for this?)

File: Jetsurvey/build.gradle [hunk 2]
```diff
@@ -71,4 +72,11 @@ subprojects {
             jvmTarget = "1.8"
         }
     }
+    configurations.configureEach {
```
Comment (chrisbanes) [inline Jetsurvey/build.gradle:75]: Add a `TODO` comment why this is needed

**Pull request merged**